src/promptflow-core/promptflow/core/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 # flake8: noqa
 from promptflow._core.tool import ToolProvider, tool
-from promptflow.core._flow import AsyncFlow, Flow
+from promptflow.core._flow import AsyncFlow, AsyncPrompty, Flow, Prompty
```

Comment:
change the prompty docstring a bit: 
from promptlfow.core import AzureOpenAIModelConfiguration
![image](https://github.com/microsoft/promptflow/assets/47586720/ae04c5c8-4dd2-4f91-b13e-0cc7e034f20f)

Reply:
fix it